scripts/detectNotUsingGitPush1by1.fsx

```diff
@@ -794,11 +815,47 @@ let GitHubApiCall(url: string) =
     client.DefaultRequestHeaders.Add("User-Agent", userAgent)
     client.DefaultRequestHeaders.Add("X-GitHub-Api-Version", xGitHubApiVersion)
 
-    (client.GetStringAsync url).Result
+    if not(String.IsNullOrEmpty accessToken) then
+        client.DefaultRequestHeaders.Add(
+            "Authorization",
+            $"token {accessToken}"
+        )
+
+    let result =
+        try
+            (client.GetStringAsync url).Result
+        with
+        | :? AggregateException as aggregateException ->
+            match aggregateException.InnerException with
+            | :? HttpRequestException as httpRequestException ->
+                match httpRequestException.StatusCode with
```

Comment:
@realmarv this is a nullable object, an ugly C# equivalent of `Option`; the best way to deal with it from F# is pass it to Option.ofObj, so `match httpRequestException.StatusCode |> Option.ofObj`, then you can use `| Some statusCode ->` so that you don't need to call `HasValue` and `Value` at all

Reply:
With this modification I get this error: `The type 'Nullable<HttpStatusCode>' does not have 'null' as a proper value. To create a null value for a Nullable type use 'System.Nullable()'.` and I don't know what to do with it

Reply:
looks good, it's what I meant

Reply:
But I get the error I mentioned